curvefs/src/client/common/common.h

```diff
@@ -64,6 +65,26 @@ std::ostream &operator<<(std::ostream &os, MetaServerOpType optype);
 
 const uint32_t MAXXATTRLENGTH = 256;
 
+const char kCurveFsWarmupXAttr[] = "curvefs.warmup.op";
+const char kCurveFsWarmupOpAdd[] = "add";
+const char kCurveFsWarmupTypeList[] = "list";
+const char kCurveFsWarmupTypeSingle[] = "single";
```

Comment:
is there anywhere uses them apart from `common.cpp`, if so, move them to cpp file.

Reply:
> is there anywhere uses them apart from `common.cpp`, if so, move them to cpp file.

just kCurveFsWarmupXAttr will be used by [FuseOpSetXattr](https://github.com/Cyber-SiKu/curve/blob/curvefs/toolv2_client_warmup/curvefs/src/client/curve_fuse_op.cpp)